com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -25,16 +17,15 @@ internal class ProbeVolumePerSceneData : MonoBehaviour, ISerializationCallbackRe
         [System.Serializable]
         struct SerializableAssetItem
         {
-            [SerializeField] public ProbeVolumeState state;
+            [SerializeField] public int state;
             [SerializeField] public ProbeVolumeAsset asset;
         }
 
-        internal Dictionary<ProbeVolumeState, ProbeVolumeAsset> assets = new Dictionary<ProbeVolumeState, ProbeVolumeAsset>();
+        internal Dictionary<int, ProbeVolumeAsset> assets = new Dictionary<int, ProbeVolumeAsset>();
```

Comment:
Being just ints, how does the user know what state is what?

Reply:
We should have a way to query names a-la layers.

Reply:
Do they need to know ? they have a name in the UI that should be enough